### PR TITLE
changelog 7.2.0

### PR DESCRIPTION
**Pull request by xlson**
/*** Grafana changelog  ***/

  ### Features / Enhancements
  * **Alerting**: Ensuring notifications displayed correctly in mobile device with Google Chat. [#27578](https://github.com/grafana/grafana/pull/27578), [@alvarolmedo](https://github.com/alvarolmedo)
  * **TraceView**: Show full traceID and better discern multiple stackTraces in span details. [#27710](https://github.com/grafana/grafana/pull/27710), [@aocenas](https://github.com/aocenas)
  * **grafana/ui**: Do not bundle jQuery. [#27667](https://github.com/grafana/grafana/pull/27667), [@kennytm](https://github.com/kennytm)

  ### Bug Fixes
  * **DataLinks**: Fixes issue with data links not interpolating values with correct field config. [#27622](https://github.com/grafana/grafana/pull/27622), [@torkelo](https://github.com/torkelo)
  * **DataProxy**: Ignore empty URL's in plugin routes. [#27653](https://github.com/grafana/grafana/pull/27653), [@domasx2](https://github.com/domasx2)
  * **Field config**: Respect config paths when rendering default value of field config property. [#27652](https://github.com/grafana/grafana/pull/27652), [@dprokop](https://github.com/dprokop)
  * **FieldConfig**: Fix mismatch in field config editor types. [#27657](https://github.com/grafana/grafana/pull/27657), [@dprokop](https://github.com/dprokop)
  * **PanelEditor**: Prevents adding transformations in panels with alerts. [#27706](https://github.com/grafana/grafana/pull/27706), [@hugohaggmark](https://github.com/hugohaggmark)
  * **StatPanel**: Fix problem where string values where always green. [#27656](https://github.com/grafana/grafana/pull/27656), [@peterholmberg](https://github.com/peterholmberg)
  * **grafana-toolkit**: avoid path.resolve with globby in moveStaticFiles. [#27670](https://github.com/grafana/grafana/pull/27670), [@kennytm](https://github.com/kennytm)

  /*** grafana-toolkit changelog  ***/

  ### Bug Fixes
  * **grafana-toolkit**: avoid path.resolve with globby in moveStaticFiles. [#27670](https://github.com/grafana/grafana/pull/27670), [@kennytm](https://github.com/kennytm)

  /*** grafana-ui changelog  ***/

  ### Features / Enhancements
  * **grafana/ui**: Do not bundle jQuery. [#27667](https://github.com/grafana/grafana/pull/27667), [@kennytm](https://github.com/kennytm)

### CODE REVIEW

**Comment by xlson** [comment]
@dprokop When we ran the generate changelog command the grafana-ui and toolkit changelog entries ended up in both the grafana changelog and their respectives sections (like you see above). We cut it out of the main grafana changelog, I presume we don't actually want that duplication?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+
+### Features / Enhancements
+* **Alerting**: Ensuring notifications displayed correctly in mobile device with Google Chat. [#27578](https://github.com/grafana/grafana/pull/27578), [@alvarolmedo](https://github.com/alvarolmedo)
+* **TraceView**: Show full traceID and better discern multiple stackTraces in span details. [#27710](https://github.com/grafana/grafana/pull/27710), [@aocenas](https://github.com/aocenas)
+
+### Bug Fixes
+* **DataLinks**: Fixes issue with data links not interpolating values with correct field config. [#27622](https://github.com/grafana/grafana/pull/27622), [@torkelo](https://github.com/torkelo)
+* **DataProxy**: Ignore empty URL's in plugin routes. [#27653](https://github.com/grafana/grafana/pull/27653), [@domasx2](https://github.com/domasx2)
+* **Field config**: Respect config paths when rendering default value of field config property. [#27652](https://github.com/grafana/grafana/pull/27652), [@dprokop](https://github.com/dprokop)
+* **FieldConfig**: Fix mismatch in field config editor types. [#27657](https://github.com/grafana/grafana/pull/27657), [@dprokop](https://github.com/dprokop)
```
Comment (hugohaggmark) [inline CHANGELOG.md:11]:
```suggestion
* **Field config**: Fix mismatch in field config editor types. [#27657](https://github.com/grafana/grafana/pull/27657), [@dprokop](https://github.com/dprokop)
```

**Comment by dprokop** [comment]
@xlson I think we don't have to include those in the main changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+* **TraceView**: Show full traceID and better discern multiple stackTraces in span details. [#27710](https://github.com/grafana/grafana/pull/27710), [@aocenas](https://github.com/aocenas)
+
+### Bug Fixes
+* **DataLinks**: Fixes issue with data links not interpolating values with correct field config. [#27622](https://github.com/grafana/grafana/pull/27622), [@torkelo](https://github.com/torkelo)
+* **DataProxy**: Ignore empty URL's in plugin routes. [#27653](https://github.com/grafana/grafana/pull/27653), [@domasx2](https://github.com/domasx2)
+* **Field config**: Respect config paths when rendering default value of field config property. [#27652](https://github.com/grafana/grafana/pull/27652), [@dprokop](https://github.com/dprokop)
+* **Field config**: Fix mismatch in field config editor types. [#27657](https://github.com/grafana/grafana/pull/27657), [@dprokop](https://github.com/dprokop)
+* **PanelEditor**: Prevents adding transformations in panels with alerts. [#27706](https://github.com/grafana/grafana/pull/27706), [@hugohaggmark](https://github.com/hugohaggmark)
```
Comment (dprokop) [inline CHANGELOG.md:12]:
```suggestion
* **Panel editor**: Prevents adding transformations in panels with alerts. [#27706](https://github.com/grafana/grafana/pull/27706), [@hugohaggmark](https://github.com/hugohaggmark)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+* **StatPanel**: Fix problem where string values where always green. [#27656](https://github.com/grafana/grafana/pull/27656), [@peterholmberg](https://github.com/peterholmberg)
```
Comment (dprokop) [inline CHANGELOG.md:13]:
```suggestion
* **Stat panel**: Fix problem where string values where always green. [#27656](https://github.com/grafana/grafana/pull/27656), [@peterholmberg](https://github.com/peterholmberg)
```

File: packages/grafana-toolkit/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+# 7.2.0 (2020-09-23)
+
+### Bug Fixes
+* **grafana-toolkit**: avoid path.resolve with globby in moveStaticFiles. [#27670](https://github.com/grafana/grafana/pull/27670), [@kennytm](https://github.com/kennytm)
```
Comment (dprokop) [inline packages/grafana-toolkit/CHANGELOG.md:4]:
```suggestion
* **grafana/toolkit**: avoid path.resolve with globby in moveStaticFiles. [#27670](https://github.com/grafana/grafana/pull/27670), [@kennytm](https://github.com/kennytm)
```

**Review by dprokop — APPROVED** [review]

**Comment by xlson** [comment]
> @xlson I think we don't have to include those in the main changelog

Where should I create an issue about improving the changelog generation tool to not include those in the main changelog?

**Comment by dprokop** [comment]
Platform backlog please @xlson

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**